udata/templates/embed-dataset.html

```diff
@@ -1,81 +1,149 @@
 <style scoped>
+    {# Following http://rscss.io/ conventions. #}
     .dataset-card {
         border: 1px solid #EEE;
-        border-bottom: 2px solid #4FA9E5;
-        max-width: {{ width }}px;
-        max-height: {{ height }}px;
         min-width: 320px;
-        min-height: 90px;
-        padding: 5px;
+        min-height: 160px;
         clear: both;
         margin: 1em;
         background: #FFF;
+        display: flex;
```

Comment:
IE11 has knows issues when using flexbox with min-height. Not sure if they'd appear here.
http://caniuse.com/#search=flex